manifests/service.pp

```diff
@@ -71,7 +71,11 @@
   Hash $service_config_hash                  = {},
   $tags                                      = [],
   $token                                     = undef,
-  Optional[Hash[String[1], String[1]]] $meta = undef,
+  Optional[Hash[
```

Comment:
thanks for the PR! What do you think about making this not optional and default to an empty hash? Would just make the datatype a bit more readable :thinking: . Please update if you think that's a good idea or just comment here. Afterwards I can merge it.

Reply:
AFAIK, making it non-optional may break older consul versions <1.1.0 (see #467).

I tried to make the datatype more readable by stretching it to multiple lines instead of a long line with the equal sign far right. Unfortunately, there is no real style guide for extensive data type definitions at class parameters.
How about just breaking into two lines?
```
Optional[Hash[String[1], Variant[
    String[1], Numeric, Boolean]]]  $meta = undef,
```

The idea behind the long data type is, to give the user the freedom to choose between using quoted and unquoted values in data yamls, but making sure only working data types can be used.

Reply:
okay, let's keep it as it is.